### PR TITLE
Dala 298 version update

### PR DESCRIPTION
**Pull request by SiegfriedSobkowiak**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] ~~At least one E2E Test exists testing the new feature~~
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [ ] ~~There is at least one picture for each story, which was created before coding has started~~
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: dataland-frontend/tests/e2e/plugins/index.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+      config.specPattern = ["tests/e2e/specs/runAll.ts"];
+      break;
+    }
+    case "preview": {
+      config.specPattern = [
+        "tests/e2e/specs/verify_deployment",
+        "tests/e2e/specs/high",
+        "tests/e2e/specs/low",
+        "tests/e2e/specs/medium",
+        "tests/e2e/specs/previsit",
```
Comment (ppuetsch) [inline dataland-frontend/tests/e2e/plugins/index.js:17]: Funktioniert das noch?

File: dataland-frontend/build.gradle.kts [hunk 2]
```diff
@@ -14,7 +14,7 @@ plugins {
 
 node {
     download.set(true)
-    version.set("16.15.1")
+    version.set("16.16.0")
```
Comment (ppuetsch) [inline dataland-frontend/build.gradle.kts:17]: Hast du Node 18.6 auch ausprobiert ob das inzwischen geht?

File: dataland-frontend/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "teste2e": "vue-cli-service test:e2e --mode development --url https://dataland-local.duckdns.org",
     "testheadless": "vue-cli-service test:e2e --mode development --url https://dataland-local.duckdns.org --headless",
     "testpipeline": "vue-cli-service test:e2e --mode development --url https://dataland-local.duckdns.org --headless",
-    "testpreview": "vue-cli-service test:e2e --url https://preview-dataland.duckdns.org --headless --config-file cypress_preview.json",
+    "testpreview": "vue-cli-service test:e2e --url https://preview-dataland.duckdns.org --headless",
```
Comment (ppuetsch) [inline dataland-frontend/package.json:13]: image

Die Updates sollen alle nicht ausgeführt werden?

File: dataland-frontend/package.json [hunk 4]
```diff
@@ -20,8 +20,8 @@
     "checkdependencies": "depcheck --ignores='@vue/cli-plugin-*,@vue/vue3-jest,sass-loader,ts-jest'"
   },
   "dependencies": {
-    "@formkit/themes": "^1.0.0-beta.8",
-    "@formkit/vue": "^1.0.0-beta.8",
+    "@formkit/themes": "^1.0.0-beta.9",
```
Comment (ppuetsch) [inline dataland-frontend/package.json:23]: kannst du nochmal npm updatepackagelock ausführen? Wenn ich das mit dieser package.json ausführe, bekomme ich andere Versionen in der package-lock

File: docker-compose.yml [hunk 5]
```diff
@@ -55,6 +55,7 @@ services:
       DATALAND_EDC_USER: ${DATALAND_EDC_USER}
       KEYCLOAK_UPLOADER_PASSWORD: ${KEYCLOAK_UPLOADER_PASSWORD}
       KEYCLOAK_READER_PASSWORD: ${KEYCLOAK_READER_PASSWORD}
+      ENVIRONMENT: ${ENVIRONMENT}
```
Comment (ppuetsch) [inline docker-compose.yml:58]: sollte sich die cypress/included version nicht auch noch im dockerfile ändern?
Aktuell noch FROM cypress/included:9.6.0

File: build.gradle.kts [hunk 6]
```diff
@@ -55,10 +55,10 @@ java.sourceCompatibility = JavaVersion.VERSION_17
 plugins {
     id("io.gitlab.arturbosch.detekt") version "1.20.0"
     id("com.github.node-gradle.node") version "3.3.0" apply false
-    id("org.springframework.boot") version "2.7.0" apply false
+    id("org.springframework.boot") version "2.7.1" apply false
```
Comment (ppuetsch) [inline build.gradle.kts:58]: gradle wrapper auf 7.5 updaten

File: dataland-frontend/package.json (hunk 3 above)
Comment (SiegfriedSobkowiak) [reply dataland-frontend/package.json:13]: grafik

das sind die die nicht funktionieren, die anderen habe ich nachgezogen

File: dataland-frontend/package.json (hunk 4 above)
Comment (SiegfriedSobkowiak) [reply dataland-frontend/package.json:23]: ist aktualisiert

File: dataland-frontend/tests/e2e/plugins/index.js (hunk 1 above)
Comment (SiegfriedSobkowiak) [reply dataland-frontend/tests/e2e/plugins/index.js:17]: ja

File: build.gradle.kts (hunk 6 above)
Comment (SiegfriedSobkowiak) [reply build.gradle.kts:58]: ist aktualisiert

File: dataland-frontend/build.gradle.kts (hunk 2 above)
Comment (SiegfriedSobkowiak) [reply dataland-frontend/build.gradle.kts:17]: probier ich mal aus

File: dataland-frontend/package.json (hunk 3 above)
Comment (ppuetsch) [reply dataland-frontend/package.json:13]: ok

File: docker-compose.yml (hunk 5 above)
Comment (ppuetsch) [reply docker-compose.yml:58]: ok

File: build.gradle.kts (hunk 6 above)
Comment (ppuetsch) [reply build.gradle.kts:58]: ok

File: dataland-frontend/package.json (hunk 4 above)
Comment (ppuetsch) [reply dataland-frontend/package.json:23]: ok

File: dataland-frontend/build.gradle.kts (hunk 2 above)
Comment (ppuetsch) [reply dataland-frontend/build.gradle.kts]: ok

**Pull request merged**